package.json

```
@@ -32,14 +33,17 @@
     "test": "jest"
   },
   "dependencies": {
-    "forta-agent": "^0.1.45"
+    "forta-agent": "^0.1.45",
+    "lru-cache": "^10.0.3"
   },
   "devDependencies": {
-    "@types/jest": "^27.0.1",
+    "@types/jest": "^27.5.2",
     "@types/nodemon": "^1.19.0",
-    "jest": "^27.0.6",
+    "forta-agent-tools": "^3.2.7",
+    "jest": "^27.5.1",
     "nodemon": "^2.0.8",
```

```suggestion
    "nodemon": "^3.0.1",
```
2.0.8 has some vulnerabilities. it's better to use the up to date version